### PR TITLE
Update README and add quickstart

### PR DESCRIPTION
**Pull request by samos123**
Our current README is too short and makes it seem like it's not ready. So adding a quickstart and some more content makes it look more ready to use.

### CODE REVIEW

**Comment by brandonjbjelland** [comment]
+1 to keeping the landing README small and linking to child docs and our landing site as much as possible.

It's reasonable to imaging the repo as a user's first exposure to substratus so there should definitely be a blurb of **what** exactly substratus but I'd rather keep more thorough user docs on the doc site and dev docs here in `/docs/`. I think we should direct users via links to those two paths. Something like:

```
[substratus-logo.png]

Your cross-cloud AI substrate.

Substratus extends the Kubernetes control plane to orchestrate the
building, training, and deployment of machine learning models.
---
Interested in trying substratus? Check out the walkthrough [here](substratus.ai/link/to/docs/site).
Rolling up your sleeves to extend substratus? Dive in [here](docs/development.md).
```

**Comment by samos123** [comment]
Agreed. Let me take another pass at this and just link to the various docs from the README. 

This was purely me seeing the README and it looking too empty to me.

**Comment by brandonjbjelland** [comment]
> Agreed. Let me take another pass at this and just link to the various docs from the README.
> 
> This was purely me seeing the README and it looking too empty to me.

Completely understandable! I had the same thought a bunch of times but mostly because the majority of readmes fall into one of two categories:

1. they give a ton of detail but also fail to explain what the project is at a high-level and why it exists. Instead these readmes are exhaustive and overly technical for a first-time visiting audience (e.g. here's how to build and compile the project from source and on every architecture). 95% of these readmes are out of date and don't work.
2. there's no effort given and the readme doesn't point you in the right direction. I don't know why the project exists or where to go first to figure it out.

I think we should find the right middleground between these on this readme.

1. what is substratus at a high-level and at a more technical level? Everyone should see this and have a basic understanding of what we're up to and if this is the project they'll find useful or interesting.
2. As a passerby (90%), user (9%), or potential contributor (1%), where should I go next to learn more?

**Review by brandonjbjelland — CHANGES_REQUESTED** [review]
https://github.com/substratusai/substratus/pull/113#issuecomment-1644416136

File: README.md [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+## Quickstart
+Read the [Quickstart Docs](https://www.substratus.ai/docs/quickstart)
+
+## Architecture
+To learn more about how Substratus works, check out the [Architecture](https://www.substratus.ai/docs/architecture) page.
+
+## Developer docs
+See docs directory
+
+
```
Comment (brandonjbjelland) [inline README.md:30]:
```suggestion

See the [development](docs/development.md) doc to get started.

```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
+## Disclaimer
+Substratus is still under active development and might introduce backward incompatible changes.
```
Comment (brandonjbjelland) [inline README.md:32]:
```suggestion
## Disclaimer

Substratus is under active development and might introduce backward incompatible changes.
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+[![Watch the video](https://img.youtube.com/vi/CLyXKJHIQ6A/hq2.jpg)](https://youtu.be/CLyXKJHIQ6A)
+
+## Docs
+Read the full docs at [substratus.ai/docs](https://www.substratus.ai/docs)
+
+## Quickstart
+Read the [Quickstart Docs](https://www.substratus.ai/docs/quickstart)
+
+## Architecture
+To learn more about how Substratus works, check out the [Architecture](https://www.substratus.ai/docs/architecture) page.
```
Comment (brandonjbjelland) [inline README.md:25]:
```suggestion
## Intro and demo

Read the [Introduction](https://www.substratus.ai/docs/introduction) and
watch the video below:
[![Watch the video](https://img.youtube.com/vi/CLyXKJHIQ6A/hq2.jpg)](https://youtu.be/CLyXKJHIQ6A)

## Docs

Read the full docs at [substratus.ai/docs](https://www.substratus.ai/docs)

## Quickstart

Read the [Quickstart Docs](https://www.substratus.ai/docs/quickstart)

## Architecture

To learn more about how Substratus works, check out the [Architecture](https://www.substratus.ai/docs/architecture) page.
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,37 @@
-# Substratus.AI
+# [Substratus.AI](https://www.substratus.ai)
 
-Your cross-cloud AI substrate: Substratus extends the Kubernetes control plane
-to orchestrate the building, training, and deployment of machine learning
-models.
+Deploy and finetune Large Language Models on K8s in a breeze.
+
+Main features:
+* Deploy large models in minutes  (e.g. latest Large Language Models)
```
Comment (brandonjbjelland) [inline README.md:6]:
```suggestion
Deploy and fine-tune Large Language Models on K8s in a breeze.

Features:

* Deploy large models in minutes  (e.g. latest Large Language Models)
```

**Review by brandonjbjelland — APPROVED** [review]
Let's do it!

**Comment by samos123** [comment]
Thanks for applying the suggestions directly and merging!

**Comment by brandonjbjelland** [comment]
Np, this is exactly along the lines I hoped to see

**Pull request merged**